pkgs/by-name/ea/earlyoom/package.nix

```diff
@@ -24,22 +23,22 @@ stdenv.mkDerivation (finalAttrs: {
 
   patches = [ ./0000-fix-dbus-path.patch ];
 
-  nativeBuildInputs = lib.optionals withManpage [
-    installShellFiles
-    pandoc
-  ];
+  nativeBuildInputs = lib.optionals withManpage [ pandoc ];
 
   makeFlags = [
     "VERSION=${finalAttrs.version}"
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
   ];
 
-  installPhase = ''
-    runHook preInstall
-    install -D earlyoom $out/bin/earlyoom
-  '' + lib.optionalString withManpage ''
-    installManPage earlyoom.1
-  '' + ''
-    runHook postInstall
+  postInstall = lib.optional withManpage ''
```

Comment:
Why?

Reply:
> Why?

Why what?
- If you mean why `optional withManpage`, it's because the `$man` output only exists when `withManpage` is enabled.
- If you mean why this `postInstall` script, it's to move man page (installed to `$out` by default) to separated output.
- If you mean why not manually install everything like the previous `installPhase`, I answered it in the previous reply.